### PR TITLE
repochecker: add --force and --limit-group options

### PR DESCRIPTION
**Pull request by lnussel**
useful for debugging to limit reviews to a specific group

### CODE REVIEW

File: repo_checker.py [hunk 1]
```diff
@@ -39,14 +39,16 @@ def __init__(self, *args, **kwargs):
...
+        self.limit_group = None
 
     def project_only(self, project, post_comments=False):
         # self.staging_config needed by target_archs().
         api = self.staging_api(project)
 
         root = ET.fromstringlist(show_results_meta(
             self.apiurl, project, multibuild=True, repository=['standard']))
-        if len(root.xpath('result[@state!="published"]')):
+        if not self.force and len(root.xpath('result[@state!="published"]')):
```
Comment (jberry-suse) [inline repo_checker.py]: I have suggest perhaps a "--skip-checks" or somesuch to skip all the checks in that section as sometimes helpful when manually running, but "--force" seems a bit ambiguous and ends up only forcing one check.

File: repo_checker.py [hunk 2]
```diff
@@ -119,6 +121,9 @@ def prepare_review(self):
                 self.logger.debug('{}: not staged'.format(request.reqid))
                 continue
 
+            if self.limit_group and group != 'openSUSE:Leap:15.0:Staging:C':
```
Comment (jberry-suse) [inline repo_checker.py]: Assume this is not intentional to have hard-coded group?

File: repo_checker.py [hunk 3]
```diff
@@ -476,6 +481,8 @@ def get_optparser(self):
         parser = ReviewBot.CommandLineInterface.get_optparser(self)
 
         parser.add_option('--skip-cycle', action='store_true', help='skip cycle check')
+        parser.add_option('--force', action='store_true', help='force review even if project is not ready')
```
Comment (jberry-suse) [inline repo_checker.py]: I have debating adding a `--skip-checks` or somesuch for skipping the normal checks on regular run. `--force` seems a bit ambiguous especially when it only skips certain checks.

File: repo_checker.py [hunk 4]
```diff
@@ -476,6 +481,8 @@ def get_optparser(self):
...
+        parser.add_option('--limit-group', metavar='GROUP', help='only review requests in specific group')
```
Comment (jberry-suse) [inline repo_checker.py]: Is this relevant for `ReviewBot` as could want to run any bot against requests in a group to get a staging finished? Although currently this is likely the only bot to be waiting on besides legal.

**Comment by jberry-suse** [comment]
Main concern being the hard-coded bit.

File: repo_checker.py (hunk 2 above)
Comment (lnussel) [reply repo_checker.py]: oh, shit :)

File: repo_checker.py (hunk 3 above)
Comment (lnussel) [reply repo_checker.py]: I don't mind, maybe I missed some cases that I didn't run into. Those should be covered by --force too of course. I already spotted the "build unchanged" case now.

File: repo_checker.py (hunk 4 above)
Comment (lnussel) [reply repo_checker.py]: could be refactored into reviewbot indeed

**Pull request merged**